packages/messaging/README.md

```diff
@@ -41,13 +41,12 @@ end
 **Gradle setup for `@adobe/react-native-aepmessaging` v1.0.0-beta.3 with in-app messaging support**  
 AEPMessaging Android SDK v1.2.0-beta.3 with in-app messaging support is published to maven snapshots. In project level build.gradle file of Android project in your RN application add `maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }` in `allprojects -> repositories`.
```

Comment:
Same here, we need to confirm if this is still correct.

Reply:
We have discussed it offline, the fix will be in another PR.